web/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
```

Comment:
There is also the [index_page](https://pub.dartlang.org/packages/index_page) package, which provides a transformer that will create simple pages like this, primarily I am just concerned about this page not being kept up to date in the future. What do you think?